content/articles/microfrontend-using-module-federation-in-solid.js/index.md

```diff
@@ -0,0 +1,292 @@
+
+There’s a lot of buzz going on around micro frontends right now. You may be asking yourself this question, how do I try it out? Not to worry. In this tutorial, we’ll be discussing micro frontends. We'll also set up micro frontends using module federation in Solid.js. Also, we take a look at how to share a micro frontend between Solid.js and React using module federation. So, let’s jump right in!
+
+### What Are Micro Frontends?
+
+Micro frontend originated from the concept of microservice. When we’re talking about micro frontends, think of it as a website composed of different web pages. These pages consist of features created by independent teams. Let’s look at Netflix for example. Imagine the header section of Netflix, the header video, and the carousel built by a separate team. Also, built by a different team is the video section. These teams have their different ways of working and they build up the main Netflix website. Another way to think of micro frontend could be separate pages on the main site. These pages combined together build the website.
+
+Micro-frontends are one popular approach to increasing flexibility. It allows teams to combine components built in different frameworks or libraries. There are several ways to put in place micro frontends. We’ll be focusing on module federation in this tutorial but first, let’s look at the benefits of micro frontend and how we split our apps.
+
+### Benefits Of Micro Frontends
+
+There are many benefits. Some of them include:
+
+- Independent deployments.
+- Independent updates and bug fixes.
+- Tech stack freedom.
+- Independent decoupled units.
+- Application split up by business domain.
+
+### How Do We Split Apps?
+
+Here are some examples of how developers split large apps:
+
+#### By Page
+
+Sometimes, having many pages opened at the same time can crash older devices. In such cases, splitting by page is the safest option. If you have good routing, you can run separate, specific micro-apps for each page. This is also beneficial to your team's developers.
+
+#### By Functionality
```

Comment:
Follow the English rules. No uppercase in the middle of the sentence? Update where applicable!